src/styles/blocks/stars-modifiers.css

```diff
@@ -0,0 +1,19 @@
+.stars--1 :nth-child(-n + 1) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url('../../images/star-active.svg');
+}
```

Comment:
You can write all selectors together, and use one line with a property  `background-image: url('../../images/star-active.svg'`